src/js/control-bar/seek-to-live.js

```diff
@@ -61,9 +56,13 @@ class SeekToLive extends Button {
    */
   updateLiveEdgeStatus(e) {
     if (this.player_.liveTracker.behindLiveEdge()) {
+      this.setAttribute('aria-disabled', false);
       this.removeClass('vjs-at-live-edge');
+      this.controlText('Seek to live, currently behind live');
     } else {
       this.addClass('vjs-at-live-edge');
+      this.controlText('Seek to live, currently playing live');
+      this.setAttribute('aria-disabled', true);
```

Comment:
do we want to use `this.disable()` and `this.enable()` here instead? https://github.com/videojs/video.js/blob/master/src/js/clickable-component.js#L147-L177 and
https://github.com/videojs/video.js/blob/master/src/js/button.js#L79-L95

Reply:
I don't know if we want the button to dim visually, which will happen if we do that. I think we just want the button to "dim" for screen readers, normal users will see a cursor change and will already be at live with a red circle.